public/mountains_WGS84.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
Shouldn't this be put in the DB instead?

Reply:
Maybe. I think that is beyond my scope, so tell me.

Reply:
For the moment I would prefer to keep it as this as it will be easier to follow modifications from Meteo France.